### ISSUE TITLE
Resnet50  output has 2048-dim feature

### ISSUE THREAD

**Issue by wangchust** [ronghuaiyang/arcface-pytorch#13]
The Resnet class in resnet.py has an attribute of 'self.fc5 = nn.Linear(512 * 8 * 8, 512)', the fisrt 512 should match the output dims of backbone conv. It is OK when use resnet18 or resnet34. But what happens if we use resnet50? Should I change the first 512 to 2048. If so, would it bring too much params?

**Comment by ronghuaiyang** [comment]
yes, you should change 512 to 2048,  the mode size will be larger, the performance will be better.
